src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.ImportBefore.targets

```diff
@@ -47,14 +47,14 @@
     </PropertyGroup>
 
     <!-- Diagnostic messages for troubleshooting -->
-    <Message Importance="low" Text="SonarQube.Integration.ImportBefore.targets was loaded" />
-    <Message Importance="low" Text="SonarQube analysis targets file found: $(AnalysisTargetsFileFound)" />
-    <Message Importance="low" Text="SonarQube analysis targets imported: $(SonarQubeTargetsImported)" />
+    <Message Importance="normal" Text="Sonar: ($(MSBuildProjectName)) SonarQube.Integration.ImportBefore.targets was loaded" />
```

Comment:
I've increased the output level of some messages to "normal" so that it doesn't require a diagnostic log to see them.
It adds at most eight additional messages in the log output, which isn't overwhelming.

I've also changed all of the messages to be prefixed with `Sonar: (MyProjectFile.csproj)` to make them easy to search for.